contracts/abstracts/ListingManager.sol

```diff
@@ -31,6 +31,9 @@ abstract contract ListingManager is Controlable {
   }
 
   function _createListing(address tokenContract, uint256 tokenId, uint256 salePrice, address seller) internal returns (uint256 listingId) {
+    require(_isBlacklistedUser(seller) == false, 'User is blacklisted');
```

Comment:
You can simplify this line (and the 2 others by replacing it by
```solidity
require(!_isBlacklistedUser(seller), 'User is blacklisted');
```
Also please add the name of the contract in the revert message, so we can more easily trace back where the error originate, like
```txt
ListingManager: User is blacklisted
```